pkg/validate/validate.go

```diff
@@ -32,70 +28,44 @@ const kubeconformCommentFormat = `
 
 </details>
 `
-const inRepoSchemaLocation = "./schemas"
 
-var localSchemasLocation string
+func getSchemaLocations(ctx context.Context, tempRepoPath string) []string {
+	locations := []string{
+		// schemas included in kubechecks
+		"default",
 
-func getSchemaLocations() []string {
-	getSchemasOnce.Do(func() {
-		ctx := context.Background()
-		_, span := otel.Tracer("Kubechecks").Start(ctx, "GetSchemaLocations")
-		schemasLocation := viper.GetString("schemas-location")
+		// schemas collected globally
+		"https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/",
```

Comment:
should we make this configurable? Maybe via env vars.

Reply:
Sorry, looks like this references the same repo as `"default"`, and doesn't need to be repeated.